### PR TITLE
Documentation example figs

### PR DESCRIPTION
**Pull request by engeir**
* Adds example figures to README, comparing default matplotlib to cosmoplots
* Sets the font family in all `set_` functions to `Times`

### CODE REVIEW

**Comment by audunth** [comment]
> 

The journals most of us are interested in (AIP, IOP) use times fonts so we should stick to that as the default.

**Comment by audunth** [comment]
> @gregordecristoforo No, I thought the other functions were implementing the same defaults, just with slight changes based on some special cases?

We should use `set_rcparams_dynamo`. It is updated to among other things handle the new default legend style correctly for .eps - figures. I would rather we make changes to `set_rcparams_dynamo` if we need the other styles.

**Comment by engeir** [comment]
I pulled back two commits, so the `set_` functions that are in `cosmoplots:main` are the same as on this branch, except from some added type hints and the added font family specification.

Also, the `set_rcparams_dynamo` is now included just as it is in `uit_scripts`.

**Comment by gregordecristoforo** [comment]
Audun told me today in person that there is no reason to keep `set_rcparams_aip` since `set_rcparams_dynamo` should be the new standard (maybe on should rename it to `set_rcparams` or `set_rcparams_standard` or something alike). Also, shall we include `log_tick_format` and the examples from uit_scripts.plotting?

**Comment by engeir** [comment]
So we only really need `set_rcparams_dynamo`, not any of the others targeted to posters etc.?

`log_tick_format == change_log_axis_base`, so it is already included. I changed the name in the hope of making it more obvious what it does.

**Comment by gregordecristoforo** [comment]
> So we only really need set_rcparams_dynamo, not any of the others targeted to posters etc.?

the functions `*_thisckline`, `*_poster` and `*_talk` haven't been tested in a while but could stay there. @audunth can you confirm this? 

> log_tick_format == change_log_axis_base, so it is already included. I changed the name in the hope of making it more obvious what it does.

Thanks for clearing this up, never mind my previous comment about this then.

**Comment by gregordecristoforo** [comment]
I just talked to Audun in person, he suggests to always use `_dynamo` and only keep the parameters in the other functions that are different to  `_dynamo()` i.e.
```Python
set_rcparams_article_thickline()
set_rcparams_poster()
```
so that `_poster` overwrites a few parameters from `_dynamo`.

File: README.md [hunk 1]
```diff
@@ -2,11 +2,25 @@
...
+git clone https://github.com/uit-cosmo/cosmoplots.git
+poetry install
+```
+
 # Use
+
 Set your `rcparams` before plotting in your code, for example:
+
 ```Python
 from cosmoplots import figure_defs
```
Comment (gregordecristoforo) [inline README.md:25]: change to `import cosmoplots
`

File: cosmoplots/figure_defs.py [hunk 2]
```diff
@@ -34,31 +34,139 @@
...
+    myParams["ytick.left"] = True
+    myParams["ytick.right"] = True
+    # All ticks point inward
+    myParams["xtick.direction"] = "in"
+    myParams["ytick.direction"] = "in"
+
+    return axes_size
+
+
+def set_rcparams_aip(
```
Comment (gregordecristoforo) [inline cosmoplots/figure_defs.py:139]: this function can be removed

File: cosmoplots/figure_defs.py [hunk 3]
```diff
@@ -87,26 +192,46 @@ def set_rcparams_aip(myParams, num_cols=1, ls="thin"):
...
+    myParams["ytick.right"] = True
     # All ticks point inward
     myParams["xtick.direction"] = "in"
     myParams["ytick.direction"] = "in"
 
     return axes_size
 
 
-def set_rcparams_article_thickline(myParams):
+def set_rcparams_article_thickline(myParams: mpl.RcParams) -> None:
```
Comment (gregordecristoforo) [inline cosmoplots/figure_defs.py:234]: keep only parameters that are different to `set_rcparamts_dynamo`

File: cosmoplots/figure_defs.py [hunk 4]
```diff
@@ -147,7 +272,7 @@ def set_rcparams_article_thickline(myParams):
     myParams["ytick.right"] = True
 
 
-def set_rcparams_poster(myParams):
+def set_rcparams_poster(myParams: mpl.RcParams) -> None:
```
Comment (gregordecristoforo) [inline cosmoplots/figure_defs.py:275]: keep only parameters that are different to `set_rcparamts_dynamo`

File: cosmoplots/figure_defs.py [hunk 5]
```diff
@@ -193,7 +320,7 @@ def set_rcparams_poster(myParams):
     myParams["ytick.right"] = True
 
 
-def set_rcparams_talk(myParams):
+def set_rcparams_talk(myParams: mpl.RcParams) -> mpl.RcParams:
```
Comment (gregordecristoforo) [inline cosmoplots/figure_defs.py:323]: keep only parameters that are different to `set_rcparamts_dynamo`

**Review by gregordecristoforo — APPROVED** [review]

**Pull request merged**